.github/workflows/graalvm.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        graalvm: ['20.2.0.java8', '20.2.0.java11']
+        graalvm: ['20.3.0.java8', '20.3.0.java11']
```

Comment:
This should be done in the [project template](https://github.com/micronaut-projects/micronaut-project-template/blob/master/.github/workflows/graalvm.yml#L17) because then it would be propagated automatically to all the modules

Reply:
Yeah probably, this also removes the predicate to skip h2 tests on Java 11

Reply:
I mean, you can leave it here if you need it for this PR, it's not a problem at all :-)

Just keep in mind that modifications of the workflows in the projects would be overridden in the future unless you also change the template project.

Reply:
Yeah that's fine, any idea why builds fail on the publish test reports stage?